src/engine/EventManager.cxx

```diff
@@ -562,14 +556,20 @@ void EventManager::checkEvents(SDL_Event &event, Engine &engine)
         else if (!tileToPlace.empty() && m_placementAllowed)
         {
           // if targetObject.size > 1 it is a tile bigger than 1x1
+            // TODO: Remove the check here, there is no need for handling this differently
```

Comment:
why adding a note instead of removing it
or we need it currently ?

Reply:
because this is still a draft. I can't finish it before the other PRs are merged, cause they changed a lot where setTileID is called.  I want to refactor a bit more here too, so I just commented it out for now